main/roles/setup_configuration/tasks/main.yml

```diff
@@ -1,8 +1,25 @@
+- name: Get data from URL
+  uri:
+    url: "https://raw.githubusercontent.com/lmc-mint/chains/master/{{ chain_id }}/.data.json"
```

Comment:
This is a point to your fork. It should be the official repository. You can handle the error or put the path of the data in a variable.